fastseq_cli/transformers_generate.py

```diff
@@ -118,6 +247,24 @@ def run_generate():
     parser.add_argument("--without_fastseq_opt", action="store_true")
     parser.add_argument("--no_repeat_ngram_size", type=int, default=None,
                          required=False, help="size of no repeat ngram")
+    parser.add_argument("--include_special_tokens", action="store_true")
+    parser.add_argument("--clean_up_tokenization_spaces", action="store_true")
+    parser.add_argument("--preprocess_cpu_num",
+                        type=int,
+                        default=2,
+                        required=False,
+                        help="pre-processing worker threads")
+    parser.add_argument("--postprocess_cpu_num",
```

Comment:
postprocess_workers

Please also update fs argument from "post-process-workers" to "postprocess-workers"

Reply:
some lightweight fs models (like wmt) have big speed gain by tuning up postproceess workers. Did we try same for hf models?

Reply:
I tried for Bart and T5 , I did not notice much difference with increasing post-process workers.

Reply:
> postprocess_workers
> 
> Please also update fs argument from "post-process-workers" to "postprocess-workers"

Done